GameServerSource/Assets/PlayFabSDK/Shared/SignalR.NET20/Http/HttpHelper.cs

```diff
@@ -180,6 +210,7 @@ private static void GetResponseCallback(IAsyncResult asynchronousResult)
             }
             catch (Exception ex)
             {
+                UnityEngine.Debug.LogFormat("Some exception occured: {0}", ex.Message);

```

Comment:
Use Debug.LogException(ex).
This hides a bunch of the information in ex, like stacktrace, exception type, etc.